Gems/ROS2/gem.json

```
@@ -1,6 +1,6 @@
 {
     "gem_name": "ROS2",
-    "version": "4.0.0",
+    "version": "4.0.1",
```

This PR adds new feature so version 4.1.0 seems more suitable 
```suggestion
    "version": "4.1.0",
```